modules/siddhi-core/src/main/java/org/wso2/siddhi/core/table/InMemoryEventTable.java

```diff
@@ -41,24 +41,26 @@
 import org.wso2.siddhi.query.api.expression.Expression;
 import org.wso2.siddhi.query.api.util.AnnotationHelper;
 
-import java.util.*;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.TreeMap;
 
 public class InMemoryEventTable implements EventTable, Snapshotable {
```

Comment:
Can we add class level Java Docs for all the classes that are currently missing them?